applications/nrf5340_audio/verification/verify.cmake

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.13.1)
+
+set(REQ_C_COMPILER_VERSION 9.2.1)
+set(REQ_C_COMPILER arm-none-eabi-gcc)
+set(DESIRED_ZEPHYR_SDK_VERSION 0.13.1)
+# Checks which compiler is used. GNU Arm Embedded Toolchain is strongly preferred.
+# https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads
+# E.g. for compiler version 10.2.1: the "10-2020-q4-major" release includes this. You have to check which
+# GNU release contains the correct compiler.
+
+if(DEFINED ENV{ZEPHYR_TOOLCHAIN_VARIANT})
+    if ($ENV{ZEPHYR_TOOLCHAIN_VARIANT} STREQUAL "gnuarmemb")
```

Comment:
it is perfectly valid for a user not to set the toolchain variant in environment, but do: `-DZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb`.

This is very convenient when working with multiple toolchains, however this code breaks such possibility.

Why are you testing directly on the environment setting here, and not the internal toolchain setting in the build system ?